.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- Thanks for contributing to OHHTTPStubs! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] -->
+
+### Checklist
+
+- [ ] I've checked that all new and existing tests pass.
+- [ ] I've updated the documentation if necessary.
```

Comment:
None of the other checklist items end with a period (`.`), so why those two? 🤔 
(I'm OK for either style… as long as it's applied consistently 😄  )